test/InterversionTests/SmugglerTests.cs

```diff
@@ -130,14 +130,31 @@ public async Task CanExportFromCurrentAndImportTo42(ExcludeOn excludeOn)
             if (excludeOn == ExcludeOn.Export)
                 exportOptions.OperateOnTypes &= ~(DatabaseItemType.Attachments | DatabaseItemType.RevisionDocuments | DatabaseItemType.CounterGroups);
             var exportOperation = await storeCurrent.Smuggler.ExportAsync(exportOptions, file);
-            
+
             await exportOperation.WaitForCompletionAsync(_operationTimeout);
 
             DatabaseStatistics expected = await storeCurrent.Maintenance.SendAsync(new GetStatisticsOperation());
 
             //Import
             var importOptions = new DatabaseSmugglerImportOptions { SkipRevisionCreation = true };
-            importOptions.OperateOnTypes &= ~DatabaseItemType.TimeSeries;
+
+            // Operate on types that will remain relevant for v4.2 
+            importOptions.OperateOnTypes &= ~(DatabaseItemType.TimeSeries | DatabaseItemType.ReplicationHubCertificates);
+            importOptions.OperateOnDatabaseRecordTypes = DatabaseRecordItemType.Client |
```

Comment:
I think it would be better to handle that in the server side. If the client API is newer than we should ignore the unknown values, if same or older and we cannot parse, then we should throw. Will discuss that with Egor and Arek tomorrow.

Reply:
done